### PR TITLE
5.0.0 The ReadMe

### PR DESCRIPTION
**Pull request by say25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ From NuGet.
 | Swashbuckle.AspNetCore version 1.0.0 - 2.5.0 | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Examples/ |
 | Swashbuckle.AspNetCore version 3.0.0  | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ |
 | Swashbuckle.AspNetCore version 4.0.0 and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 4.5.1 |
-| Swashbuckle.AspNetCore version 5.0.0-beta and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 5.0.0-beta |
+| Swashbuckle.AspNetCore version 5.0.0 and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 5.0.0 |
```
Comment (mattfrear) [inline README.md]: Please revert, -beta is correct

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ navigating to swagger/v1/swagger.json
 
 ![swagger.json](https://mattfrear.files.wordpress.com/2016/01/capture.jpg)
 
-As of version 5.0.0-beta, XML examples are also supported.
+As of version 5.0.0, XML examples are also supported.
```
Comment (mattfrear) [inline README.md]: Please revert, -beta is correct

File: README.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ You'll see some more realistic data (or whatever you want):
 
 ![response with awesome data](https://mattfrear.files.wordpress.com/2015/04/response-new.png?w=700&h=358)
 
-As of version 5.0.0-beta, XML examples are also supported.
+As of version 5.0.0, XML examples are also supported.
```
Comment (mattfrear) [inline README.md]: Please revert, -beta is correct

Comment (say25) [reply README.md]:
```suggestion
As of version 5.0.0-beta, XML examples are also supported.
```

File: README.md (hunk 2 above)
Comment (say25) [reply README.md]:
```suggestion
As of version 5.0.0-beta, XML examples are also supported.
```

File: README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ From NuGet.
 | Swashbuckle.AspNetCore version 1.0.0 - 2.5.0 | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Examples/ |
 | Swashbuckle.AspNetCore version 3.0.0  | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ |
 | Swashbuckle.AspNetCore version 4.0.0 and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 4.5.1 |
-| Swashbuckle.AspNetCore version 5.0.0-beta and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 5.0.0-beta |
+| Swashbuckle.AspNetCore version 5.0.0 and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 5.0.0 |
```
Comment (say25) [inline README.md]:
```suggestion
| Swashbuckle.AspNetCore version 5.0.0-beta and above | https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/ Version >= 5.0.0-beta |
```

File: README.md (hunk 1 above)
Comment (say25) [reply README.md]: Sounds good just didn’t know if it was intended to advertise beta or not in the docs

**Comment by mattfrear** [comment]
Nice one, thanks for the PR.

**Pull request merged**